Update the 1.7 release notes to include MediaFieldSettings enhancements (#14254)
OrchardCMS/OrchardCore: src/docs/releases/1.7.0.md

### New `Media Indexing` Feature

The Lucene and Elasticsearch feature are capable of searching within `PDF` files because `PDF` files are indexed by default. As an enhancement, we added a new feature called `Media Indexing` which extends the media indexing capability to also encompass searching within files with the following extensions `.txt`, `.md`, `.docx`, and `.pptx`.

### `Media` Module

> When enabling both the `Media` and `Content Fields` features, you can attach a media field to your content. This feature enables you to link different types of media to your content. As an enhancement, we added a new property to `MediaFieldSettings` named `AllowedExtensions`, to allow you to define extensions, thereby restricting the permissible media file extensions for the field.

Reviewer: ```suggestion
When enabling both the `Media` and `Content Fields` features, you can attach a media field to your content. This feature enables you to link different types of media to your content. As an enhancement, we added a new property to `MediaFieldSettings` named `AllowedExtensions`, to allow you to define extensions, thereby restricting the permissible media file extensions for the field.
```
```suggestion
When enabling both the `Media` and `Content Fields` features, you can attach a `MediaField` to your content. This feature enables you to link different types of media to your content. As an enhancement, we added a new property to `MediaFieldSettings` named `AllowedExtensions`, to allow you to define extensions, thereby restricting the permissible media file extensions for the field.
```

> When enabling both the `Media` and `Content Fields` features, you gain the capability to attaching a Media Field with your content. This feature enables you to link different types of media to your content. As an enhancement, we added a new property to `MediaFieldSettings` named `AllowedExtensions`, to allow you to define extensions, thereby restricting the permissible media file extensions for the field.

Reviewer: ```suggestion
When enabling both the `Media` and `Content Fields` features, you can attach a Media Field with your content. This feature enables you to link different types of media to your content. As an enhancement, we added a new property to `MediaFieldSettings` named `AllowedExtensions`, to allow you to define extensions, thereby restricting the permissible media file extensions for the field.
```

!!! info

>     The specified extensions within the `MediaFieldSettings` must be a subset of the globally allowed extensions that are defined in `AllowedFileExtensions` within `MediaOptions`.

Reviewer: ```suggestion
    The specified extensions within the `MediaFieldSettings` must be a subset of the globally allowed extensions defined in `AllowedFileExtensions` within `MediaOptions`.
```
